src/ui/StepManager.js

```diff
@@ -118,6 +152,9 @@ StepManager.prototype.stepOverForward = function () {
     return
   }
   var step = this.traceManager.findStepOverForward(this.currentStepIndex)
+  if (this.solidityMode) {
```

Comment:
This just rounds it back to the original value most of the time. We need to actually go to the next step and skip any call that was made in the meantime.